include/kindyn/robot.hpp

```diff
@@ -126,6 +126,29 @@ namespace cardsflow {
                 ROS_WARN_STREAM_THROTTLE(1, "writing virtual, "
                         "you probably forgot to implement your own write function?!");
             };
+
+            /**
+             * Setter functions for gymFunctions class.
+             */
+            void setMotorState(int index1, int index2, double value);
+
+            void setJointState(int index1, int index2, double value);
+
+            void setIntegrationTime(double dt);
+
+            /**
+             * Getter functions for gymFunctions class
+             */
+            virtual double getLimit(int index1, int index2) {};
```

Comment:
Here again it is not clear what index1 and index2 stand for

Reply:
is it now?